README.md

```diff
@@ -1,21 +1,39 @@
-# 1Password Helm Charts
+<!-- Image sourced from https://blog.1password.com/introducing-secrets-automation/ -->
+<img alt="" role="img" src="https://blog.1password.com/posts/2021/secrets-automation-launch/header.svg"/>
 
-This repository hosts the official 1Password Helm Charts.
+<header style="text-align: center;">
+	<h1 style="margin-top: 20px; margin-bottom:0; border-bottom: none;">1Password Helm Charts</h1>
+	<p>This repository hosts the official 1Password Helm Charts.</p><br>
+	<a href="/QUICKSTART.md">
+		<img alt="Get started" src="https://user-images.githubusercontent.com/45081667/226940040-16d3684b-60f4-4d95-adb2-5757a8f1bc15.png" height="37"/>
+	</a>
+</header>
```

Comment:
Two changes I've made here compared to the template:
- Added `border-bottom: none;` so that the description looks like a subtitle.
- Added a `<br>` to add extra spacing between the description and the `Get started` button.

Let me know if these adjustments make sense.

Reply:
According to [GitHub Markup process](https://github.com/github/markup#github-markup), in-line styling is ignored as part of the sanitization of the HTML. 😞

Therefore, this comment can be resolved since in-line styling doesn't work.